tests/test_browsers_playlists.py

```diff
@@ -111,8 +111,8 @@ class TPlaylistIntegration(TestCase):
 
     def setUp(self):
         quodlibet.config.init()
-        self.lib = quodlibet.browsers.search.library = FileLibrary()
-        quodlibet.browsers.search.library.librarian = SongLibrarian()
+        self.lib = quodlibet.browsers.tracklist.library = FileLibrary()
```

Comment:
Definitely better! 

But yes I do slightly prefer `quodlibet.browsers.tracks.library`... also because the songslist might one day get renamed tracks list (we had a discussion ages ago around _tracks_ > _songs_ naming-wise I think)

Reply:
I'll change that. Thanks for the review!  =)